x-pack/legacy/plugins/task_manager/server/task_store.ts

```diff
@@ -427,18 +427,13 @@ export class TaskStore {
   }
 }
 
+// function changed to stop sorting by _id: https://github.com/elastic/kibana/issues/52517
```

Comment:
I think we should leave a note here and on the fetch() method in task_manager.ts that pagination is likely not reliable because we do not have a unique value per document for use with search_after.  

Pagination doesn't seem to be used at the moment, but we should leave hints for people that they shouldn't expect it to work if they try in the future.  Maybe open & reference an issue, that we can address if/when pagination is ever needed (for example if we add a task management UI, which is a possibility).

Reply:
I don't know what the original thinking here was, perhaps @tsullivan can shed some light, but as it seems to refer to pagination and we don't currently use pagination for anything in TM, it's probably best to just remove all references to pagination and if one day we need it, we can figure out how best to support it.

No?